specification/sigma-rules-specification.md

```diff
@@ -182,13 +183,14 @@ related:
 
 Currently the following types are defined:
 
-* `derived`: The rule was derived from the referred rule or rules, which may remain active.
-* `obsolete`: The rule obsoletes the referred rule or rules, which aren't used anymore.
-* `merged`: The rule was merged from the referred rules. The rules may still exist and are in use.
-* `renamed`: The rule had previously the referred identifier or identifiers but was renamed for whatever
+- `correlation`: The rule is used in the referred correlation rule.
```

Comment:
Unsure if we should add this. As it's a backreference to an existing reference from the correlation rule I see various drawbacks:

* It requires to edit the rules referred from correlation rules.
* Because it's redundant information, this will likely end up in inconsistencies.
* If the rule is referred from a private rule the reference makes no sense.
* All other `related` types denote rule management aspects while this one denotes rule usage.

I don't see it as very useful because I can't trust that the list is maintained properly. On the other side it's quite easy to output it from pySigma or a rule management tool.